store/cassandra/store_cassandra.go

```diff
@@ -33,12 +33,12 @@ const Month_sec = 60 * 60 * 24 * 28
 const Table_name_format = `metric_%d`
 
 var (
-	errChunkTooSmall  = errors.New("unpossibly small chunk in cassandra")
-	errStartBeforeEnd = errors.New("start must be before end.")
-	errReadQueueFull  = errors.New("the read queue is full")
-	errReadTooOld     = errors.New("the read is too old")
-	errTableNotFound  = errors.New("table for given TTL not found")
-	errCtxCanceled    = errors.New("context canceled")
+	errChunkTooSmall = errors.New("unpossibly small chunk in cassandra")
```

Comment:
shouldn't that be `impossibly`? is `unpossibly` a word?

Reply:
yes. no.